src/main/java/uk/gov/justice/hmpps/offenderevents/services/PrisonApiService.java

```diff
@@ -29,7 +29,19 @@ public PrisonerDetails getPrisonerDetails(final String offenderNumber) {
 
 }
 
-record PrisonerDetails(String legalStatus, boolean recall) {
+record PrisonerDetails(LegalStatus legalStatus, boolean recall) {
 }
 
 
+enum LegalStatus {
+    RECALL,
+    DEAD,
```

Comment:
`DECEASED` shurely?

Reply:
These are prison-api enums. I'm hoping not many dead people get received into prison ...